converters/fromZigbee.js

```diff
@@ -1201,11 +1201,22 @@ const converters = {
             }
         },
     },
+    ZNCLDJ11LM_ZNCLDJ12LM_curtain_options_output: {
+        cluster: 'genBasic',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options) => {
+            console.log(`genBasic ${JSON.stringify(msg.data)}`);
+            if (msg.data['1025']) {
+                return {options: msg.data['1025']}
```

Comment:
Please convert this to something human friendly like https://github.com/Koenkk/zigbee-herdsman-converters/pull/835/files#diff-3e0c275bd41af699153bf59d8dda4a2aR762